src/lib/utils/numericUuid.ts

```diff
@@ -0,0 +1,4 @@
+function numericUuid() {
+    return Math.round(Math.random() * 10000000000000000);
```

Comment:
The previous one generated a string where the interface and the functions that were using it wanted a number.